schemas/context/enduserids.example.1.json

```diff
@@ -2,8 +2,8 @@
   "https://ns.adobe.com/experience/mcid": {
     "@id": "https://ns.adobe.com/entities/identity/mcid123",
     "xdm:namespace": {
-      "@id": "https://ns.adobe.com/entities/namespace/11111",
-      "xdm:code": "AA11111"
+      "@id": "https://ns.adobe.com/entities/namespace/AVID",
```

Comment:
Is this a real customer name? Would it make sense to replace it with a non-protected name?

Reply:
AVID is the registered name in our "namespace issuing authority" and stands for "Analytics Visitor Identity" It is used like this on almost all report suites that have migrated to use the Visitor Identity Service. So it is accurate.

Reply:
Ok.